
  \documentclass[twoside]{article}
  \usepackage[affil-it]{authblk}
  \usepackage{lipsum} % Package to generate dummy text throughout this template
  \usepackage{eurosym}
  \usepackage[sc]{mathpazo} % Use the Palatino font
  \usepackage[T1]{fontenc} % Use 8-bit encoding that has 256 glyphs
  \usepackage[utf8]{inputenc}
  \linespread{1.05} % Line spacing-Palatino needs more space between lines
  \usepackage{microtype} % Slightly tweak font spacing for aesthetics\[IndentingNewLine]
  \usepackage[hmarginratio=1:1,top=32mm,columnsep=20pt]{geometry} % Document margins
  \usepackage{multicol} % Used for the two-column layout of the document
  \usepackage[hang,small,labelfont=bf,up,textfont=it,up]{caption} % Custom captions under//above floats in tables or figures
  \usepackage{booktabs} % Horizontal rules in tables
  \usepackage{float} % Required for tables and figures in the multi-column environment-they need to be placed in specific locations with the[H] (e.g. \begin{table}[H])
  \usepackage{hyperref} % For hyperlinks in the PDF
  \usepackage{lettrine} % The lettrine is the first enlarged letter at the beginning of the text
  \usepackage{paralist} % Used for the compactitem environment which makes bullet points with less space between them
  \usepackage{abstract} % Allows abstract customization
  \renewcommand{\abstractnamefont}{\normalfont\bfseries} 
  %\renewcommand{\abstracttextfont}{\normalfont\small\itshape} % Set the abstract itself to small italic text\[IndentingNewLine]
  \usepackage{titlesec} % Allows customization of titles
  \renewcommand\thesection{\Roman{section}} % Roman numerals for the sections
  \renewcommand\thesubsection{\Roman{subsection}} % Roman numerals for subsections
  \titleformat{\section}[block]{\large\scshape\centering}{\thesection.}{1em}{} % Change the look of the section titles
  \titleformat{\subsection}[block]{\large}{\thesubsection.}{1em}{} % Change the look of the section titles
  \usepackage{fancyhdr} % Headers and footers
  \pagestyle{fancy} % All pages have headers and footers
  \fancyhead{} % Blank out the default header
  \fancyfoot{} % Blank out the default footer
  \fancyhead[C]{X-meeting $\bullet$ November 2017 $\bullet$ S\~ao Pedro} % Custom header text
  \fancyfoot[RO,LE]{} % Custom footer text
  %----------------------------------------------------------------------------------------
  % TITLE SECTION
  %---------------------------------------------------------------------------------------- 
 
 \title{\vspace{-15mm}\fontsize{24pt}{10pt}\selectfont\textbf{ Functional prediction of stress-modulated proteins of Deinococcus radiodurans }} % Article title
  
  
  \author{ Ricardo Valle Ladewig Zappala$^{1}$, Manuela Leal da Silva$^{2}$, Pedro Geraldo Pascutti$^{1}$, Claudia de Alencar Santos Lage$^{1}$, }
  
  \affil{ 1 Universidade Federal do Rio de Janeiro

2 Instituto Nacional de Metrologia Qualidade e Tecnologia

 }
  \vspace{-5mm}
  \date{}
  
  %---------------------------------------------------------------------------------------- 
  
  \begin{document}
  
  
  \maketitle % Insert title
  
  
  \thispagestyle{fancy} % All pages have headers and footers
  %----------------------------------------------------------------------------------------  
  % ABSTRACT
  
  %----------------------------------------------------------------------------------------  
  
  \begin{abstract}
  The Deinococcaceae group comprises some of the robust known extremophilic bacteria. Attempts have specially focused on responses against extreme doses of gamma radiation or desiccation to explain survival of Deinococcus radiodurans against them. Many defensive mechanisms were shown to exist in D. radiodurans, and transcriptomes already performed in response to gamma radiation and desiccation revealed that some genes were transcribed to proteins of undefined functions, while others have never been expressed under those conditions. Therefore, it is expected that such genes with unknown functions could code for novel resistance proteins to those extreme conditions. The present study aims to identify and perform function prediction for hypothetical, unique proteins of D. radiodurans, without similarity to any other known protein. Sequences of a group of 26 proteins, with 23 expressed in D. radiodurans after radiation or desiccation stresses were retrieved, which hypothetical functions were predicted by the best scores after BLAST alignments and CD-search. Information about the proteins was gathered through alignments against Uniprot and PDB databases. Using molecular modeling tools as I-TASSER, SWISS MODEL and MODELLER, 3D models were built for all hypothetic proteins and they were mainly evaluated by Ramachandranâ€˜s Plot, RMSD and DOPE score. The best models were then submitted to structural classification on SCOP and CATH servers. This approach enabled us to better infer about the function of twenty candidates for new extremophilic proteins, of which thirteen went through comparative modeling with multiple templates. Three seemed to belong to the group of intrinsically disordered proteins, and three have not aligned to any proper templates by comparative modeling. Among the seven predicted proteins using structures from SWISS MODEL are: DR0438, a DNA binding protein; DR1263, a N-glycosidase; DR1314, a photosystem-like transmembrane protein; DR1370, a structural lipoprotein; DR2073, a kinase; and DR2441, an acetyl-transferase. Among the 26 analyzed proteins, the most interesting one appears to be the DR0491 gene product, showing 25\% identity, 41\% similarity, and covering 90\% of the sequence correspondent to the Escherichia coli heat shock protein Hsp31. This may represent an essential role on catalysis of damaged proteins, as well as proper folding assistance on other unstable proteins. After several steps of investigation, modeling and structural analyses, complementary tools such as phylogeny, molecular dynamics and molecular docking were also performed to strengthen the significance of the observed results, and this particular resistance toolbox with novel and exclusive proteins was referred as the "Black Box Genome of D. radiodurans".
  
  Funding: CNPq \\ 
  \end{abstract}
  \end{document} 